bananas_api/new_upload/session_validation.py

```diff
@@ -94,8 +102,14 @@ def validate_packet_size(session, package):
     size += 16  # md5sum
     size += len(session.get("dependencies", [])) * 4
     size += 1
-    for tag in session.get("tags", package.get("tags", [])):
-        size += len(tag) + 2
+    for key, value in Classification().dump(session.get("classification")).items():
+        size += len(key) + 2
```

Comment:
This does not match what bananas-server is doing. It sends "value" for str-values, and "key" for bool-values.

Reply:
Yup. And yes, this is fine ;)

This already checks for the future, that we can always send `{key}:{value}` or `{key}:yes`. The bananas-server implementation also checks if this fits. This way, nobody will introduce a package where we cant send key/value, if we ever want to send the full classification to the client.

In reality, it currently only sends either `key` or `value`, which is always smaller than the combination.

In other words: it does match the validation of bananas-server, and it always sends something smaller as tags.